.github/workflows/ci.yml

```
@@ -0,0 +1,340 @@
+        key: memory-${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+    
+    - name: Build with debug symbols
+      run: cargo build --profile dev
+    
+    - name: Run memory leak detection
+      run: |
+        # Create a simple test runner for memory leak detection
+        cat > memory_test.rs << 'EOF'
+        use rt::terminal::Terminal;
+        use std::time::Duration;
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)